src/sentry/issues/related/trace_connected.py

```diff
@@ -72,7 +74,90 @@ def trace_connected_issues(event: Event | GroupEvent) -> tuple[list[int], dict[s
         {
             datum["issue.id"]
             for datum in query.process_results(results[0])["data"]
-            if datum["issue.id"] != group.id  # Exclude itself
+            if datum["issue.id"] != exclude_group_id  # Exclude itself
         }
     )
-    return transformed_results, meta
+    return transformed_results
+
+
+def _trace_connected_issues_eap(
+    trace_id: str,
+    organization: Organization,
+    projects: list[Project],
+    exclude_group_id: int,
+) -> list[int]:
+    """EAP implementation of trace_connected_issues."""
+    start, end = default_start_end_dates()  # Today to 90 days back
+    snuba_params = SnubaParams(
+        start=start,
+        end=end,
+        organization=organization,
+        projects=projects,
+        environments=[],
+    )
+
+    try:
+        result = Occurrences.run_table_query(
```

Comment:
Looking at the raw EAP results in the logs, the relevant group IDs are included in the response multiple times (based on how many occurrences exist for those group IDs). Ideally, we would aggregate into distinct group IDs EAP-side, not Python-side. Ask EAP team how to modify the query in order to do this.